bootstrap/pkg/utils/k8utils.go

```diff
@@ -314,6 +341,18 @@ func (a *Apply) namespace(namespace string) error {
 					string(kftypes.NAMESPACE), namespace, nsErr),
 			}
 		}
+	} else {
```

Comment:
Thanks for your PR
for above branch creating a new namespace, we should also make sure the katibMetricsCollectorLabel added. I think you can add the label in line 335

Reply:
@hougangliu Thanks, looks like I missed it in the commit.